region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -1136,6 +1268,24 @@
       "gModeRegainMobility": {},
       "note": "Wait 2 minutes for a global Geemer."
     },
+    {
+      "link": [6, 7],
+      "name": "Carry Shinecharge",
+      "entranceCondition": {
+        "comeInShinecharged": {
+          "framesRequired": 120
+        }
+      },
+      "requires": [
+        "canShinechargeMovementComplex",
+        "canDownBack"
```

Comment:
Changed it to a notable strat now. After trying this out some more, the thing I found that seems to make it much easier is to walk off the ledge rather than running off. This costs just one extra shinecharge frame but makes it so you can just aim down, shoot twice, then hold left against the floating platform; no need to move back and forth.